com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -113,62 +113,75 @@ public UnityRLInitParameters Initialize(CommunicatorInitParameters initParameter
                     {
                         RlInitializationOutput = academyParameters
                     },
-                    out input);
-
-                var pythonPackageVersion = initializationInput.RlInitializationInput.PackageVersion;
```

Comment:
Moved this logic outside of the `try`. It no longer throws (`throw new UnityAgentsException("ICommunicator.Initialize() failed.");`) and instead returns false.